### PR TITLE
Rewrite documentation

### PR DESCRIPTION
**Pull request by nck-mlcnv**
- [x] Write the markdown documentation
- [x] ~~Copy it to the wiki~~ (will do when main branch is updated)
- [x] Update the owl file

### CODE REVIEW

File: docs_new/introduction.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# IGUANA
```
Comment (bigerl) [inline docs_new/introduction.md:1]: try to keep duplication as minimal as possible. Ideally this says them same as the README.md

In that case, it should be a symlink to README.md not a copy.

File: example-suite.yml [hunk 2]
```diff
@@ -1,23 +1,28 @@
 datasets:
```
Comment (bigerl) [inline example-suite.yml:1]:
```suggestion
# This file showcases the configuration of most IGUANA features.

datasets:
```

File: schema/iguana.owl [hunk 3]
```diff
@@ -30,155 +30,168 @@
...
+    </owl:Class>
+
+    <owl:Class rdf:about="&iont;metric/PQPS">
+        <rdfs:label>NoQ Metric</rdfs:label>
+        <rdfs:comment xml:lang="en">Penalized Queries Per Second.</rdfs:comment>
+        <rdfs:subClassOf rdf:resource="&iont;Metric" />
+    </owl:Class>
+
+    <owl:Class rdf:about="&iont;metric/PAvgQPS">
+        <rdfs:label>NoQ Metric</rdfs:label>
```
Comment (bigerl) [inline schema/iguana.owl:137]: labels are wrong

**Review by bigerl — CHANGES_REQUESTED** [review]
thx for the PR. Looks good and almost done. 
Please find my suggestions and comments below.

File: docs_new/configuration/metrics.md [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+- `NoQ`
+- `QMPH`
+- `NoQPH`
+- `PQPS`
+- `PAvgQPS`
+- `AES`
+- `EachQuery`
+
+### QPS
+Queries per second.
```
[the remark this answers is not in the document]
Comment (nck-mlcnv) [reply docs_new/configuration/metrics.md:53]: I've switched to a table.

**Review by bigerl — APPROVED** [review]
Looks good now. Thx for the work 👍🏻

**Pull request merged**